spec/client_spec.rb

```diff
@@ -628,6 +626,41 @@
     end
   end
 
+  describe '#organization' do
```

Comment:
I was trying to do this on [this branch's file](https://github.com/stormpath/stormpath-sdk-ruby/blob/add-organizations-delegate/spec/client_spec.rb). Can you add all of the organization specs that I wrote there to see if they pass? They should, but they were not passing for me for some reason.